### ISSUE TITLE
Crash When Attempting to Load Non-Existent Video

### ISSUE THREAD

**Issue by MadokaAyukawa** [wangqr/Aegisub#72]
AS will ask if I want to load the accompanying video when I load a subtitle file. I often get subtitles from other subbers so my video may not have the same name or be under the same directory structure. Most of the time, if the file is not found, AS will tell me so, but I've come across instances where the non-existent video will cause AS to crash to the desktop.

One such case occurs when the video is entered in the subtitle file as:
```
Audio File: Arimura Kasumi no Satsukyu S01E01v2 720p HDTV AAC H.264-NSBC.mkv
Video File: Arimura Kasumi no Satsukyu S01E01v2 720p HDTV AAC H.264-NSBC.mkv
```
I have a different video under a different name. I realize that some of the error is on me, but it would be preferable if AS just told me it can't find it rather than crashing to the desktop.

**Comment by wangqr** [comment]
Can you remove the AviSynth.dll in aegisub folder (so that AviSynth provider won't be used) and see if there's any difference?

**Comment by MadokaAyukawa** [comment]
That seemed to prevent the crash.

**Comment by wangqr** [comment]
Yeah I recently found AviSynth+ is causing some crashes. I've tried 3.7.0 for a while and it seems the issue has been fixed.

To upgrade AviSynth+:
1. Download `AviSynthPlus_3.7.0_20210111-filesonly.7z` from https://github.com/AviSynth/AviSynthPlus/releases/tag/v3.7.0
2. Extract `AviSynthPlus_3.7.0_20210111-filesonly\x86-64\AviSynth.dll` and `AviSynthPlus_3.7.0_20210111-filesonly\x86-64\plugins\DirectShowSource.dll` to Aegisub folder

**Comment by MadokaAyukawa** [comment]
Ok, that seems to have fixed it. Thank you. I was testing r9185.

I was wondering where I can find this:

`Win64 nightly builds available at GHA artifact`

As I'd like to try the new version with the added information in the Video Box.

**Comment by wangqr** [comment]
Click the green tick after the commit, and select Details after GitHub Action:
![img](https://user-images.githubusercontent.com/8182182/114317242-aa9d4d00-9af6-11eb-83de-0b4474363adf.png)

The artifact can be found at top right corner:
![img](https://user-images.githubusercontent.com/8182182/114317281-cf91c000-9af6-11eb-83d4-22d5b54417b1.png)

**Comment by MadokaAyukawa** [comment]
Thank you! I was almost there; I didn't see the artifacts in the upper right corner.

**Comment by MadokaAyukawa** [comment]
I just tired the artifact above, but replacing those two files above still results in a crash.
